src/tools/run-make-support/src/lib.rs

```diff
@@ -164,6 +164,9 @@ pub fn set_host_rpath(cmd: &mut Command) {
 /// crate::impl_common_helpers!(CommandWrapper);
 ///
 /// impl CommandWrapper {
+///     /// Get the [`Output`][::std::process::Output] of the finished process.
+///     pub fn output(&mut self) -> Output { /* ... */ }
+///
```

Comment:
The signature of the required `output` function should probably go before, in the "and an `output` function" sentence, because this code snippet is trying to illustrative what the expansion is (here `output` is *required*, not *provided*).

Reply:
Maybe something like this

```rs
/// Example usage:
///
/// ```ignore (illustrative)
/// struct CommandWrapper { cmd: Command }// <- required `cmd` field
///
/// impl CommandWrapper {
///     /// Get the [`Output`][::std::process::Output] of the finished process.
///     pub fn output(&mut self) -> Output { /* ... */ } // <-- required `output()` method
/// }
///
/// crate::impl_common_helpers!(CommandWrapper);
///
/// impl CommandWrapper {
///     // ... additional specific helper methods
/// }
/// ```
```

to highlight that `output()` is a *required* method?

Reply:
Sure, sounds good. Added.